src/components/CVideo/CVideo.js

```diff
@@ -102,6 +103,12 @@ const getStaticStyle = (config) => {
   return style;
 };
 
+const uuid = () => v4();
+
+const getUniqueKey = () => {
```

Comment:
You can remove this method, no need for component name in key, `uuid` is enough.